README.md

```diff
@@ -1 +1,27 @@
-# MakeOfficeHours
\ No newline at end of file
+# MakeOfficeHours
+
+[![linting: pylint](https://img.shields.io/badge/linting-pylint-yellowgreen)](https://github.com/pylint-dev/pylint)
+
+# api
+
+A simple flask api server that handels students enqueue and dequeue
```

Comment:
```suggestion
A simple flask api server that handles students enqueue and dequeue
```

Reply:
This is describing the main (backend) service in the core readme; it's what people will first see when learning about this project, so it should be informative, correct, and not belittling of the project.

```suggestion
A Flask API server that handles enqueuing and dequeuing students from the office hours queue.
```

(Or similar, since I don't know the full scope of this services' responsibilies.)